src/Pages/Errors.php

```diff
@@ -59,7 +59,27 @@ public function tlms_showWarnings(): void
 
         if (! empty($this->talentlmsAdminErrors)) {
             foreach ($this->talentlmsAdminErrors as $message) {
-                echo '<div class="error notice is-dismissible">' . $message . '</div>';
+                echo '<div class="error notice is-dismissible">' . wp_kses($message, array('strong'=>array()), array('http', 'https')) . '</div>';
+            }
+        }
+    }
+
+    /**
+     * Show warnings (only on Dashboard, Setup and Integrations pages)
+     * @return void
+     */
+    public function tlms_ManualShowWarnings(){
+        require_once(ABSPATH . 'wp-admin/includes/screen.php');
```

Comment:
Why do we need this?

Reply:
We need this to call below get_current_screen()

Reply:
a few line above we call get_current_screen and we don't require this

Reply:
It is required earlier in the stack by WP, but since we call manual show warnings from the backend we need to require this specifically